<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GraficasMatplotlib</class>
 <widget class="QMainWindow" name="GraficasMatplotlib">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>150</y>
      <width>691</width>
      <height>381</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="grafica_uno">
     <property name="spacing">
      <number>6</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
    </layout>
   </widget>
   <widget class="QPushButton" name="browse">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>30</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Browse</string>
    </property>
   </widget>
   <widget class="QLabel" name="file">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>30</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Select a file:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="filename">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>30</y>
      <width>241</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="startplot">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>550</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Plot</string>
    </property>
   </widget>
   <widget class="QLabel" name="choicefrequency">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>80</y>
      <width>191</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Indicates the sampling frequency:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="frequency">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>80</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="Hertz">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>80</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 11pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>Hz</string>
    </property>
   </widget>
   <widget class="QLabel" name="choicenumchannels">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>80</y>
      <width>171</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Indicates the number of channels:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="number_channels">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>80</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="simulation">
    <property name="geometry">
     <rect>
      <x>604</x>
      <y>550</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Simulate a Path</string>
    </property>
   </widget>
   <widget class="QLabel" name="choice_time_lapse">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>120</y>
      <width>161</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Choose the time interval to view:</string>
    </property>
   </widget>
   <widget class="QLabel" name="set_min_time">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>120</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>from</string>
    </property>
   </widget>
   <widget class="QLabel" name="set_max_time">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>120</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>to</string>
    </property>
   </widget>
   <widget class="QLabel" name="Seconds">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>120</y>
      <width>47</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;DejaVu Serif&quot;;</string>
    </property>
    <property name="text">
     <string>s</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="min_time">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>120</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="max_time">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>120</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
